cl/search/tests.py

```diff
@@ -4160,3 +4173,159 @@ def test_docket_number_suffixes_query(self) -> None:
         expected = 2
         self.assertEqual(actual, expected)
         self.assertIn("Hong Liu", r.content.decode())
+
+    def test_stemming_disable_search(self) -> None:
+        """Test docket_number with suffixes can be found."""
```

Comment:
This is good. I found myself wondering though what happens when you combine phrase and exact search. Could you do this, for example: `""Inform" people" and have it find:

```
Inform people
```

But not:
```
Information people
```
And not:
```
Inform those people
```

? 

This is pretty edge-case, but I'm curious, and it would affect how we document the feature. (Probably we just need to add something to the documentation that says "Exact searches cannot be placed inside phrase searches."

Reply:
Yeah, the query `""Inform" people"` doesn't work, it seems that it just nullifies the `exact` quotes.

![Screenshot 2023-06-01 at 17 35 39](https://github.com/freelawproject/courtlistener/assets/486004/c4af69fb-4952-477b-8d18-50dc670ccaf4)

But it seems the result you expected can be achieved using just a normal phrased search, it considers the "exact" query and the phrase query at the same time.

![Screenshot 2023-06-01 at 17 41 07](https://github.com/freelawproject/courtlistener/assets/486004/415d2904-8694-45c7-b869-b713255e89f4)
![Screenshot 2023-06-01 at 17 40 59](https://github.com/freelawproject/courtlistener/assets/486004/b2a49901-2d20-4d56-8d00-9383284fc653)

Yeah, we should document this kind of search.  I just added a comment in #2779 to remember it.

I added a test related to this case.